config/api_config.go

```diff
@@ -78,3 +78,8 @@ func (cfg *APIConfig) Load(v *viper.Viper) error {
 	cfg.readIn(v)
 	return v.Unmarshal(&cfg)
 }
+
+// APIEndpoint returns the full URL for a given endpoint
+func APIEndpoint(endpoint string) string {
```

Comment:
This will work as long as people are not trying to test something locally. If they've overridden the base url in their local config to work against a development server, this won't pick it up.